src/views/Main/components/styles.module.css

```diff
@@ -102,13 +102,17 @@
 }
 
 .navButton {
+  display: flex;
```

Comment:
We'll have to be a little careful with IE11, but it looks manageable :). http://caniuse.com/#feat=flexbox